config/owasp/suppressions.xml

```diff
@@ -1477,6 +1477,48 @@
     </suppress>
     <suppress>
         <notes><![CDATA[
+   file name: spring-webmvc-5.2.2.RELEASE.jar
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.springframework/spring\-webmvc@.*$</packageUrl>
+        <cpe>cpe:/a:pivotal_software:spring_framework</cpe>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: spring-webmvc-5.2.2.RELEASE.jar
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.springframework/spring\-webmvc@.*$</packageUrl>
+        <cve>CVE-2020-5397</cve>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: spring-webmvc-5.2.2.RELEASE.jar
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.springframework/spring\-webmvc@.*$</packageUrl>
+        <cve>CVE-2020-5398</cve>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: spring-test-5.2.2.RELEASE.jar
+   ]]></notes>
+        <sha1>444e2f91f667558df1853b15915ac28b2185404c</sha1>
+        <cpe>cpe:/a:pivotal_software:spring_framework</cpe>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: spring-test-5.2.2.RELEASE.jar
+   ]]></notes>
+        <sha1>444e2f91f667558df1853b15915ac28b2185404c</sha1>
+        <cve>CVE-2020-5397</cve>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: spring-test-5.2.2.RELEASE.jar
+   ]]></notes>
+        <sha1>444e2f91f667558df1853b15915ac28b2185404c</sha1>
+        <cve>CVE-2020-5398</cve>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
```

Comment:
do we need all these suppressions?

Reply:
They are the suppressions from the Dependency-Check and are needed to suppress:

1. file name: spring-webmvc-5.2.2.RELEASE.jar
2. file name: spring-test-5.2.2.RELEASE.jar